extensions/amp-ad-network-adsense-impl/0.1/adsense-a4a-config.js

```diff
@@ -21,69 +21,41 @@
+import {
+  getExperimentBranch,
+  forceExperimentBranch,
+  randomlySelectUnsetExperiments,
+} from '../../../src/experiments';
+import {dev} from '../../../src/log';
 
 /** @const {!string}  @private */
 export const ADSENSE_A4A_EXPERIMENT_NAME = 'expAdsenseA4A';
 
-// The following experiment IDs are used by Google-side servers to
-// understand what experiment is running and what mode the A4A code is
-// running in.  In this experiment phase, we're testing 8 different
-// configurations, resulting from the Cartesian product of the following:
-//   - Traditional 3p iframe ad rendering (control) vs A4A rendering
-//     (experiment)
-//   - Experiment triggered by an external page, such as the Google Search
-//     page vs. triggered internally in the client code.
-//   - Doubleclick vs Adsense
-// The following two objects contain experiment IDs for the first two
-// categories for Adsense ads.  They are attached to the ad request by
-// ads/google/a4a/traffic-experiments.js#googleAdsIsA4AEnabled when it works
-// out whether a given ad request is in the overall experiment and, if so,
-// which branch it's on.
-
-// We would prefer the following constants to remain private, but we need to
-// refer to them directly in amp-ad-3p-impl.js and amp-a4a.js in order to check
-// whether we're in the experiment or not, for the purposes of enabling
-// debug traffic profiling.  Once we have debugged the a4a implementation and
-// can disable profiling again, we can return these constants to being
-// private to this file.
-/**
- * const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const ADSENSE_A4A_EXTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH = {
-  control: '117152652',
-  experiment: '117152653',
-};
+/** @type {string} */
+const TAG = 'amp-ad-network-adsense-impl';
 
-export const ADSENSE_A4A_EXTERNAL_DELAYED_EXPERIMENT_BRANCHES_PRE_LAUNCH = {
-  control: '117152654',
-  experiment: '117152655',
+/** @const @enum{string} */
+export const ADSENSE_EXPERIMENT_FEATURE = {
+  HOLDBACK_EXTERNAL: '2092618',
+  HOLDBACK_INTERNAL: '2092616',
+  DELAYED_REQUEST: '117152655',
 };
 
-/**
- * const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const ADSENSE_A4A_EXTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH = {
-  control: '2092617',
-  experiment: '2092618',
-};
-
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const ADSENSE_A4A_INTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH = {
-  control: '117152670',
-  experiment: '117152671',
-};
-
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const ADSENSE_A4A_INTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH = {
-  control: '2092615',
-  experiment: '2092616',
+/** @const @type {!Object<string,?string>} */
+export const URL_EXPERIMENT_MAPPING = {
+  '-1': MANUAL_EXPERIMENT_ID,
+  '0': null,
+  // Holdback
+  '1': '2092617',
```

Comment:
can you either add the strings to the enum, or add comments for each explaining what they are?